src/collision/CollisionManager.js

```diff
@@ -1,56 +1,88 @@
 import .CollisionChecker;
 import .ColliderMap;
 
-/**
-  * Responsible for updating and managing collision checks.
-  * Collisions are registered as 2 things, `a` and `b`. The check is run every tick to see if
-  * `a` and `b` have collided.
-  * {@link CollisionManager#stop}, {@link CollisionManager#start}, {@link CollisionManager#remove}
-  * are exposed in case you want to control the collisions manually; however, the Collision Manager is
-  * smart about running collisions. If a group, `a` or `b`, is a single {@link Actor} or an array of
-  * {@link Actor}s, then the collision manager will automatically remove the check when actors in question
-  * are destroyed.
-  * @class CollisionManager
-  */
+/** @lends CollisionManager */
 exports = Class(function(supr) {
 
+  /**
+   * Responsible for updating and managing collision checks.
+   * Collisions are registered as 2 things, `a` and `b`. The check is run every tick to see if
```

Comment:
This is a confusing sentence, try something like "Collisions checks are registered for both subjects, `a` and `b`."